src/Parser/MistweaverMonk/CPM_ABILITIES.js

```diff
@@ -0,0 +1,52 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+
+import ISSUE_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
+
+export const SPELL_CATEGORY = {
+  ROTATIONAL: 'Rotational Spell',
+  COOLDOWNS: 'Cooldown',
+  OTHERS: 'Spell',
+};
+
+const CPM_ABILITIES = [
```

Comment:
This looks like it contains all the main spells which is great. With Disc Priests I started also including utility spells in this list to show a more complete overview of the available arsenal of each spec. It could be nice to include that here too if you want (example: https://github.com/MartijnHols/WoWAnalyzer/blob/master/src/Parser/DisciplinePriest/CPM_ABILITIES.js#L126)